dashboard/src/components/status/Status.vue

```diff
@@ -19,7 +19,12 @@
 	<div class="Status">
     <div class="is-size-1">Wolfia Shard Status</div>
 		<div class="stafflist columns is-centered is-multiline" :class="{ 'is-loading': !shardsLoaded }">
-      <Shard />
+      <div
+        v-for="shard in shards"
+				:key="shard"
```

Comment:
This should probably be the shard ID
I can't really explain what it does exactly but the Vue docs are insisting that its important: https://vuejs.org/v2/guide/list.html#Maintaining-State